README.md

```diff
@@ -6,12 +6,21 @@
 
 A GitHub Action for installing, configuring and running hardware-accelerated Android Emulators on macOS virtual machines.
 
-The old ARM-based emulators were slow and are no longer supported by Google. The modern Intel Atom (x86 and x86_64) emulators require hardware acceleration (HAXM on Mac & Windows, QEMU on Linux) from the host to run fast. This presents a challenge on CI as to be able to run hardware accelerated emulators within a docker container, **KVM** must be supported by the host VM which isn't the case for cloud-based CI providers due to infrastructural limits. If you want to learn more about this, here's an article I wrote: [Running Android Instrumented Tests on CI](https://dev.to/ychescale9/running-android-emulators-on-ci-from-bitrise-io-to-github-actions-3j76).
+The old ARM-based emulators were slow and are no longer supported by Google. The modern Intel Atom (x86 and x86_64) emulators can be fast, but rely on two forms of hardware acceleration to reach their peak potential: [Graphics Acceleration](https://developer.android.com/studio/run/emulator-acceleration#accel-graphics), e.g. `emulator -accel on` and [Virtual Machine(VM) Acceleration](https://developer.android.com/studio/run/emulator-acceleration#accel-vm), e.g. `emulator -gpu host`. Note: GPU and VM Acceleration are two different and non-mutually exclusive forms of Hardware Acceleration.
```

Comment:
I think the examples given are reversed?

Reply:
@mattjohnsonpint Thank you so much for catching this!! I really appreciate it. Addressed in `8f64fa8`

Reply:
Looks great now.  Thanks!